### DISCUSSION TITLE
I would like to know how to properly end the demonstration collection process.

### DISCUSSION THREAD

**Discussion by tkdggg** [isaac-sim/IsaacLab#593] [Q&A]
### Describe the bug

Isaac Lab documentation->Getting Started->Running Existing Scripts->Imitation Learning
https://isaac-sim.github.io/IsaacLab/source/setup/sample.html#imitation-learning

When i run  
"./isaaclab.sh -p source/standalone/workflows/robomimic/collect_demonstrations.py --task Isaac-Lift-Cube-Franka-IK-Rel-v0 --num_envs 1 --num_demos 10 --device keyboard ",  
no matter how I operate the keyboard and change the num_demos, the HDF5 file it generates is always only 96 bytes.

I would like to know how to properly end the demonstration collection process.

### System Info

- Isaac Sim Version: 4.0
- OS: Ubuntu 20.04
- CUDA: 12.2
- GPU Driver: 535.183.01

**Comment by Dhoeller19** [comment]
Hi @tkdggg,
In the script, the data is flushed on success. In the example there, you have to pick up the cube and then place it on the target. Then there will be a terminal output confirming a trajectory has been collected.

**Reply by qiangl9618** [reply]
@Dhoeller19 
Hi, I collected 200 demo data in the mock example, grabbed the object and put it in the specified position, And then use " ./isaaclab.sh -p source/standalone/workflows/robomimic/train.py --task Isaac-Lift-Cube-Franka-IK-Rel-v0 --algo bc --dataset logs/robomimic/Isaac-Lift-Cube-Franka-IK-Rel-v0/hdf_dataset.hdf5 " to do the training, after the completion of the training, always unable to achieve the expected effect.
I would like to know if you have experimented with examples of imitation learning, and could you share your experience
